R/landing_popup_module.R

```diff
@@ -66,7 +66,7 @@ landing_popup_module <- function(label = "Landing Popup",
   )
   checkmate::assert_multi_class(buttons, classes = c("shiny.tag", "shiny.tag.list"))
 
-  logger::log_info("Initializing landing_popup_module")
+  message("Initializing landing_popup_module")
```

Comment:
Finally! Dude, I love it. Let me check on Monday in depth actual `teal.logger` solution

Reply:
:)